README.md

```diff
@@ -2,92 +2,86 @@
 
 [![Build Status](https://travis-ci.com/cisagov/assessment-data-import-terraform.svg?branch=develop)](https://travis-ci.com/cisagov/assessment-data-import-terraform)
 
-## About ##
-
 This project creates the resources used to import assessment data into AWS.
 
-## Requirements ##
-
-| Name | Version |
-|------|---------|
-| terraform | ~> 0.12.0 |
-| aws | ~> 2.0 |
+## Pre-requisites ##
 
-* [AWS CLI access
-  configured](
-  https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
-  on your system
-* [Terraform installed](
-  https://learn.hashicorp.com/terraform/getting-started/install.html)
-  on your system
+- [Terraform](https://www.terraform.io/) installed on your system.
+- AWS CLI access
+  [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html)
+  for the appropriate account on your system.
+- An accessible AWS S3 bucket to store Terraform state
+  (specified [here](backend.tf)).
```

Comment:
Put these files names between backticks, please.

Reply:
Looking for clarification here. Do you mean change `[here](backend.tf)` to ``[`backend.tf`](backend.tf)``? It's used as a link in that line.

Reply:
Yep, we're converting from the practice of linking "here" to actual text, i.e. `backend.tf` inside the brackets.

Reply:
It's a Plain Language best-practice, iirc.

Reply:
@jsf9k Please see 20ca527 for the changes in question. @hillaryj Thanks for confirming my suspicion. I'd assumed that was what he meant, but was looking for confirmation ^^

Reply:
Heh, I actually misread the links the first time.  I like the changes, though.

I was wrong, but somehow that led to an improvement.  Praise Fortuna!